package.json

```diff
@@ -11,8 +11,8 @@
     "url": "git@github.com:Skyscanner/babel-plugin-transform-i18n.git"
   },
   "engines": {
-    "node": ">=16.14.2",
-    "npm": ">=8.5.0"
+    "node": "^18.16.0",
```

Comment:
We should also ideally keep the `>=` as per: https://github.com/Skyscanner/babel-plugin-transform-i18n/pull/9 to avoid warnings when running npm install.

Reply:
Updated. Thanks!